packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -110,7 +144,9 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
                 <ShellPane collapsed={this.state.tableOfContentsCollapsed}>
                     {this.getPaneCollapseToggle()}
                     {this.getSlotItems(this, ShellSlot.pane)}
-                    {this.getRootToc(this.props.children, SiteSlot.category, path, "/")}
+                    <ul className={this.props.managedClasses.site__paneToc}>
+                        {this.getRootToc(this.props.children, SiteSlot.category, path, "/")}
+                    </ul>
                 </ShellPane>
                 <ShellCanvas>
                     <ShellActionBar>
```

Comment:
can we fix the spacing on this?

Reply:
won't fix

Reply:
Following up as I told Jeff to won't fix this: The spacing in question is correct in context.